views/includes/entity.html

```diff
@@ -21,14 +21,20 @@
 {% endset %}
 
 {% if pageParams.isRegistration %}
-{% set entityNameLink = CREATE_CHANGE_LINK(
-                           OE_CONFIGS.ENTITY_CHANGE_NAME,
-                           "Name of the overseas entity",
-                           "change-entity-name-button")
-%}
+  {% set entityNameLink = CREATE_CHANGE_LINK(
+    OE_CONFIGS.ENTITY_CHANGE_NAME,
```

Comment:
strange not to use changeLinkUrl here as well as line 31

Reply:
not really, because there is no existing variables to make up the change link easily for register journey.
They've just gone with `ENTITY_CHANGE_NAME` which is `register-an-overseas-entity/overseas-name/#entity_name`. 
So using `changeLinkUrl` would append the baseUrl again. And I didn't want to go changing it too much to be honest since a couple of controllers use this view.

Reply:
ok, just checking